### PR TITLE
Fix bug in CodeLines

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- LineClass is null in some cases

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 1]
```diff
@@ -12,11 +12,18 @@
...
     string foldableClass = Model.CodeLine.LineClass;
-    string [] foldableClassParts = Regex.Split(foldableClass, "\\s+");
-    bool isHeading = foldableClassParts.Any(x => x.EndsWith("-heading"));
-    bool isContent = foldableClassParts.Any(x => x.EndsWith("-content"));
-    string hiddenSectionClass = isContent ? "d-none" : "";
-    string sectionHeadingClass = isHeading ? "border-top" : "";
+    bool isHeading = false;
+    bool isContent = false;
+    string hiddenSectionClass = String.Empty;
+    string sectionHeadingClass = String.Empty;
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:18]: Is this class used anywhere?

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 2]
```diff
@@ -12,11 +12,16 @@
...
-    bool isHeading = foldableClassParts.Any(x => x.EndsWith("-heading"));
-    bool isContent = foldableClassParts.Any(x => x.EndsWith("-content"));
-    string hiddenSectionClass = isContent ? "d-none" : "";
-    string sectionHeadingClass = isHeading ? "border-top" : "";
+    bool isHeading = false;
+    bool isContent = false;
+    string hiddenSectionClass = String.Empty;
+    if (!String.IsNullOrWhiteSpace(foldableClass))
+    {
+        string [] foldableClassParts = Regex.Split(foldableClass, "\\s+");
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:20]: Why does this need to be a regex split, verse a simple string.split with removeemptyentries? Generally string split will be more efficient than regex but I'm not sure how performance critical this code is.

Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:20]: Ther is no particular reason am using Regex.Split. String.Split will still throw a null exception in this case though.

Comment (weshaggard) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:20]: Yes sure you still need to fix the issue you were trying to fix no matter which split option you use.

**Review by praveenkuttappan — APPROVED** [review]

**Comment by praveenkuttappan** [comment]
Since it's critical to have this merged to unblock, I am merging it. @chidozieononiwu : You can submit follow up PR to fix any comments on this PR.

**Pull request merged**